site/content/docs/manifest/environment.en.md

```diff
@@ -261,6 +261,9 @@ The name of an existing S3 bucket in which to store the access logs.
 <span class="parent-field">http.public.access_logs.</span><a id="http-public-access-logs-prefix" href="#http-public-access-logs-prefix" class="field">`prefix`</a> <span class="type">String</span>   
 The prefix for the log objects.
 
+<span class="parent-field">http.public.ingress.</span><a id="http-public-ingress-source-ips" href="#http-public-ingress-source-ips" class="field">`source_ips`</a> <span class="type">Boolean</span>    
+Restrict ingress traffic for the public load balancer to come from source IPs.
```

Comment:
1. Is the type a boolean or a `[]string`? 😝 
2. Can we include an example?
3. Do we not need to first add documentation for just `http.public.ingress` before the `source_ips`?
4. Can we remove from the docs the deprecated fields and instead move them under `http.public.ingress`

Reply:
1. 👻  
2. Yes 
3. Wanxian is working on the UX fix docs so she will add the rest of the fields. 
4. Good point. @Lou1415926 what do you think about this?